### DISCUSSION TITLE
2010/05/24/google%E6%94%AF%E6%8C%81https%E5%8A%A0%E5%AF%86%E6%90%9C%E7%B4%A2

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#290] [Announcements]
# 2010/05/24/google%E6%94%AF%E6%8C%81https%E5%8A%A0%E5%AF%86%E6%90%9C%E7%B4%A2

对于中国用户来说，HTTPS的加密搜索一劳永逸地解决了中国用户对于Google搜索的稳定性问题。在Google中搜索某些敏感词语，会出现“连接被重置”，这导致一些很常用的词语，例如“学习”、“胡萝卜”、“温度计”等无法在Google搜索，而现在，使用HTTPS的Google，我们可以搜索你想搜索的任何词语， 再也不会出现“连接被重置”了。

https://blog.qizhen.xyz/2010/05/24/google%E6%94%AF%E6%8C%81https%E5%8A%A0%E5%AF%86%E6%90%9C%E7%B4%A2

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2010年08月4日 14:37 编辑
Google加密搜索的另一种解决方案http://www.williamlong.info/archives/2271.html

回复
Jing
2010年05月30日 22:15 编辑
不行了

回复
毅
2010年05月25日 09:29 编辑
有用！

回复
l
2010年05月24日 18:02 编辑
估计很快这个证书就会被封锁，和googledocs一样，不过在这之前还是先用着吧。

回复
